theme/themes/pxt/globals/site.variables

```diff
@@ -149,7 +149,7 @@
 @serialGraphColor: black;
 @editorCloseColor: black;
 @editorCloseColorHover: darken(@editorCloseColor, 25%);
-@editorCloseBorderSize: 2px;
+@serialConsoleFont: 'Monaco', 'Menlo', 'Ubuntu Mono', 'Consolas', 'source-code-pro', monospace;
```

Comment:
Set the default here to be @blocklySvg. Let's not repeat this list if we dont have to.

Reply:
Leaving this alone, as per our discussion.